ladybug/comfort/standard/pmv.py

```diff
@@ -0,0 +1,406 @@
+            ppd_comfort_thresh
+            humid_ratio_upper
+            humid_ratio_low
+            still_air_threshold
+        """
+
+        self._ppd_thresh = ppd_comfort_thresh or 10
+        self._hr_upper = humid_ratio_upper or 1
+        self._hr_lower = humid_ratio_lower or 0
+        self._still_thresh = still_air_threshold or 0.1
+
+        assert 5 <= self._ppd_thresh <= 100, \
+            'ppd_comfort_thresh must be between 5 and 100. Got {}'.format(
+                self._ppd_thresh)
+        assert 0 <= self._hr_upper <= 1, \
+            'humid_ratio_upper must be between 0 and 1. Got {}'.format(
+                self._hr_upper)
+        assert 0 <= self._hr_lower <= 1, \
+            'humid_ratio_lower must be between 0 and 1. Got {}'.format(
+                self._hr_lower)
+        assert 0 <= self._still_thresh, \
+            'still_air_threshold must be greater than 0. Got {}'.format(
+                self._still_thresh)
+        assert self._hr_lower <= self._hr_upper, \
+            'humid_ratio_lower must be less than humid_ratio_upper. {} > {}'.format(
+                self._hr_lower, self._hr_upper)
+
+    @property
+    def ppd_comfort_thresh(self):
+        """The threshold of the percentage of people dissatisfied (PPD) beyond which
+        the conditions are not acceptable.  The default is 10%.
+        """
+        return self._ppd_thresh
+
+    @property
+    def humid_ratio_upper(self):
+        """A number representing the upper boundary of humidity ratio above which conditions
+        are considered too humid to be comfortable.  The default is set to 0.03 kg
+        wather/kg air.
+        """
+        return self._hr_upper
+
+    @property
+    def humid_ratio_lower(self):
+        """A number representing the lower boundary of humidity ratio below which conditions
+        are considered too dry to be comfortable.  The default is set to 0 kg wather/kg
+        air.
+        """
+        return self._hr_lower
+
+    @property
+    def still_air_threshold(self):
+        """A number representing the wind speed beyond which the formula for Standard
+        Effective Temperature (SET) is used to dtermine PMV/PPD (as opposed to Fanger's
+        original equation). The default is set to 0.1 m/s.
+        """
+        return self._still_thresh
+
+    def is_comfortable(self, ppd, humidity_ratio):
+        """Determine if conditions are comfortable or not."""
+        return True if (ppd <= self._ppd_thresh and
+                        humidity_ratio >= self._hr_lower and
+                        humidity_ratio <= self._hr_upper) else False
+
+    def thermal_condition(self, pmv, ppd):
+        """Determine whether conditions are cold, neutral or hot."""
+        if ppd >= self._ppd_thresh:
+            return 1 if pmv > 0 else -1
+        else:
+            return 0
+
+    def discomfort_reason(self, pmv, ppd, humidity_ratio):
+        """Determine if conditions are comfortable or not."""
```

Comment:
missing documentation. What does -2, 2 and 0 means.

It looks like the docstring for `thermal_condition` and `discomfort_reason` are swapped.

Reply:
Ah, wait. I see you were pointing to the other class. I just fixed this.